Project2/presentation/detail/pages/InfoController.swift

```diff
@@ -0,0 +1,236 @@
+        NSLayoutConstraint.activate([
+            releaseDateLabel.leftAnchor.constraint(equalTo: view.leftAnchor,constant: 30),
+            releaseDateLabel.topAnchor.constraint(equalTo: view.topAnchor,constant:  70),
+            
+            releaseDate.topAnchor.constraint(equalTo: releaseDateLabel.bottomAnchor, constant: 8),
+            
+            releaseDate.leftAnchor.constraint(equalTo: releaseDateLabel.leftAnchor),
+            
+            releaseDate.rightAnchor.constraint(equalTo: releaseDateLabel.rightAnchor),
+        ])
+        
+        NSLayoutConstraint.activate([
+            languageLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: 70),
+            languageLabel.leftAnchor.constraint(equalTo: releaseDateLabel.rightAnchor, constant: 50),
+
+            language.topAnchor.constraint(equalTo: languageLabel.bottomAnchor, constant: 8),
+
+            language.leftAnchor.constraint(equalTo: languageLabel.leftAnchor),
+
+            language.rightAnchor.constraint(equalTo: languageLabel.rightAnchor, constant: -20),
+        ])
+        
+        NSLayoutConstraint.activate([
+            budgetLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: 70),
+            budgetLabel.leftAnchor.constraint(equalTo: languageLabel.rightAnchor, constant: 50),
+
+            budget.topAnchor.constraint(equalTo: budgetLabel.bottomAnchor, constant: 8),
+            budget.leftAnchor.constraint(equalTo: budgetLabel.leftAnchor),
+            budget.rightAnchor.constraint(equalTo: budgetLabel.rightAnchor),
+        ])
+
+        NSLayoutConstraint.activate([
+            revenueLabel.topAnchor.constraint(equalTo: budget.bottomAnchor,constant: 3),
+            revenueLabel.leftAnchor.constraint(equalTo: budgetLabel.leftAnchor),
+
+            revenue.topAnchor.constraint(equalTo: revenueLabel.bottomAnchor, constant: 5),
+            revenue.leftAnchor.constraint(equalTo: revenueLabel.leftAnchor)
+        ])
+        
+        NSLayoutConstraint.activate([
+            overviewLabel.topAnchor.constraint(equalTo: revenue.bottomAnchor, constant: 5),
+            overviewLabel.leftAnchor.constraint(equalTo: releaseDateLabel.leftAnchor),
+        ])
+
+        NSLayoutConstraint.activate([
+            overview.topAnchor.constraint(equalTo: overviewLabel.bottomAnchor, constant: 15),
+            overviewContainerView.leftAnchor.constraint(equalTo: overviewLabel.leftAnchor),
+            overviewContainerView.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -20),
+        ])
+        
+        NSLayoutConstraint.activate([
+            overview.topAnchor.constraint(equalTo: overviewContainerView.topAnchor),
+            overview.leftAnchor.constraint(equalTo: overviewContainerView.leftAnchor),
+            overview.rightAnchor.constraint(equalTo: overviewContainerView.rightAnchor),
+        ])
+    }
+    
+    private func getDetails(){
+        viewModel.getMovieDetails(id: movie?.id ?? -1) { result in
```

Comment:
Swift-ში არის memory management სისტემა - ARC (https://docs.swift.org/swift-book/documentation/the-swift-programming-language/automaticreferencecounting/)

იმისთვის, რომ არ მოხდეს მეხსიერების leak, weak/unowned რეფერენსები უნდა გამოიყენო ფუნქციის ბლოკებში - იხილე სტატია